tools/c2chapel/test/enum.h

```diff
@@ -0,0 +1,32 @@
+
+/* Header file for enum test script */
+/* Enum without values */
+enum 
+{
+  TEST_STATUS_PASSED,
+  TEST_STATUS_FAILED
+};
+
+/* Enum with values */
+enum 
+{
+  PASSED = 1,
+  FAILED = 2
+};
```

Comment:
Let's also add a test case like this:
```
enum direction {
  NORTH,
  SOUTH,
  EAST,
  WEST
};
```

I think it would also be good if the comment before this group of constants mentioned the name (e.g. ``direction``).